05-services.md

```diff
@@ -379,6 +379,8 @@ expressed in the short form.
     service.
   - `service_completed_successfully`: specifies that a dependency is expected to run
     to successful completion before starting a dependent service.
+- `required`: when `false` Compose only warns you when the dependency service isn't started or available. If not defined
```

Comment:
Nit: maybe an empty line such as in 374 to separate the new field description more from the rest?